### PR TITLE
Feat settings concurrent updates

### PR DESCRIPTION
**Pull request by seavan**
#### Relevant info and issue/PR links 

https://app.clubhouse.io/peerio/story/12811/sdk-make-saving-settings-concurrent-safe 

#### Testing instructions  

check that saving user settings still works fine.

----
### Repository owner

- [ ] Was tested and can be merged.
- [ ] PR name follows conventional changelog format.

[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)

### CODE REVIEW

File: src/ui/settings/components/Preferences.jsx [hunk 1]
```diff
@@ -47,7 +49,6 @@ class Preferences extends React.Component {
 
     onUnreadChatSorting(ev) {
         chatStore.unreadChatsAlwaysOnTop = ev.target.checked;
-        User.current.saveSettings();
```
Comment (anri-asaturov) [inline src/ui/settings/components/Preferences.jsx]: ?

Comment (seavan) [reply src/ui/settings/components/Preferences.jsx]: @anri-asaturov it's saved in prefs. it has not relation to server user settings.

**Comment by seavan** [comment]
@ebilg ready for testing

**Comment by ebilg** [comment]
can be merged

**Pull request merged**